manifests/f/ForgQi/biliup-rs/0.1.15/ForgQi.biliup-rs.installer.yaml

```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.2 $debug=NVS1.CRLF.7-3-2.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ForgQi.biliup-rs
+PackageVersion: 0.1.15
+InstallerType: zip
+ReleaseDate: 2022-10-03
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
The package installs, but requires this dependency to run:
![image](https://user-images.githubusercontent.com/114952565/218515492-4fa8f03f-06e3-4e70-b457-d18bc996cb37.png)